uiowa/uiowa
# Config Split cleanup and Neuroscience updates

Resolves #4266 

# How to test

Review the [file changes](https://github.com/uiowa/uiowa/pull/4326/files) and check that there are no configuration files added or removed from:
* `config/sites/admissions.uiowa.edu`.
* `config/sites/brand.uiowa.edu`
* `config/sites/grad.uiowa.edu`
